src/src/Helpers/Rules_calculate.cpp

```diff
@@ -281,12 +281,12 @@ ESPEASY_RULES_FLOAT_TYPE RulesCalculate_t::apply_quinary_operator(char op,
   const UnaryOperator qu_op = static_cast<UnaryOperator>(op);
 
   if (UnaryOperator::Map == qu_op || UnaryOperator::MapC == qu_op) {
-    ret = mapADCtoFloat(first, second, third, fourth, fifth);
 
     // Clamp the result if the operator is MapC
     if (qu_op == UnaryOperator::MapC) {
-      ret = std::clamp(ret, std::min(fourth, fifth), std::max(fourth, fifth));
+      first = constrain(first, second, third);
```

Comment:
Hm, I think you still need the `min()`/`max()` applied here, as `second` can be `>` `third`.

Reply:
I think you are right. But this also tells me, that it is time to sleep :) Good night

Reply:
It can also be simplified like this:
```c++
if (second < third) {
  first = constrain(first, second, third);
} else {
  first = constrain(first, third, second);
}
```